cpp/src/toppra/constraint/joint_torque/pinocchio.hpp

```diff
@@ -22,6 +23,15 @@ class Pinocchio : public JointTorque {
     typedef _Model Model;
     typedef typename _Model::Data Data;
 
+    /// Build a JointTorque constraint with no friction
+    /// \param urdfFilename path to a URDF file.
+    static Pinocchio fromURDF (const std::string& urdfFilename)
+    {
+      Model model;
+      pinocchio::urdf::buildModel(urdfFilename, model);
+      return Pinocchio(model, Vector::Zero(model.nv));
```

Comment:
I realize now this is not correct as the return object will keep a reference to a deleted model. Please don't merge.